docs/demos/opc-thermometer-demo.md

```diff
@@ -83,98 +83,98 @@ Now, we must create some OPC UA PLC Servers to discover. Instead of starting fro
 
 2. (Optional) If you are using security, place the OpcPlc certificate and the CA certificate as below.
 
-   ```
-   plc
-   ├── own
-   │   ├── certs
-   │   │   └── OpcPlc [hash].der
-   │   └── private
-   │       └── OpcPlc [hash].pfx
-   └── trusted
-      ├── certs
-      │   └── someCA.der
-      └── crl
-         └── someCA.crl
-   ```
+```
+plc
+├── own
+│   ├── certs
+│   │   └── OpcPlc [hash].der
+│   └── private
+│       └── OpcPlc [hash].pfx
+└── trusted
+   ├── certs
+   │   └── someCA.der
+   └── crl
+      └── someCA.crl
+```
 3. (A) If you are not using security, copy and paste the contents below into the YAML file.
 
-   ```yaml
-   apiVersion: apps/v1
-   kind: Deployment
-   metadata:
-     name: opcplc
-     labels:
-       app: opcplc
-   spec:
-     selector:
-       matchLabels:
-         app: opcplc
-   template:
-     metadata:
-       labels: 
-         app: opcplc
-         name: opc-plc-server
-     spec:
-       hostNetwork: true
-       containers:
-       - name: opcplc1
-         image: mcr.microsoft.com/iotedge/opc-plc:latest
-         ports:
-         - containerPort: 50000
-         args: ["--pn=50000", "--aa", "--fn=1", "--ft=uint", "--ftl=65", "--ftu=85", "--ftr=True", "--sph"]
-       - name: opcplc2
-         image: mcr.microsoft.com/iotedge/opc-plc:latest
-         ports:
-         - containerPort: 50001
-         args: ["--pn=50001", "--aa", "--fn=1", "--ft=uint", "--ftl=65", "--ftu=85", "--ftr=True", "--sph"]
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: opcplc
+  labels:
+    app: opcplc
+spec:
+  selector:
+    matchLabels:
+      app: opcplc
+  template:
+    metadata:
+      labels: 
+        app: opcplc
+        name: opc-plc-server
+    spec:
+      hostNetwork: true
+      containers:
+      - name: opcplc1
+        image: mcr.microsoft.com/iotedge/opc-plc:latest
+        ports:
+        - containerPort: 50000
+        args: ["--pn=50000", "--aa", "--fn=1", "--ft=uint", "--ftl=65", "--ftu=85", "--ftr=True", "--sph", "--ut"]
+      - name: opcplc2
+        image: mcr.microsoft.com/iotedge/opc-plc:latest
+        ports:
+        - containerPort: 50001
+        args: ["--pn=50001", "--aa", "--fn=1", "--ft=uint", "--ftl=65", "--ftu=85", "--ftr=True", "--sph", "--ut"]
```

Comment:
Can we expand these arguments to their full names if possible so easier to read?

Reply:
For example `--unsecuretransport` instead of `--ut` ([params here](https://github.com/Azure-Samples/iot-edge-opc-plc))